.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - ci-*
```

Comment:
Is this part of the workflow? I haven’t seen how this gets used exactly.

Reply:
It's not part of _this_ workflow because I can't rename a branch after a PR is pushed (and having a `/` in the tag name screws up Docker tagging) but you can se it in action on this PR: https://github.com/OrcaCollective/OpenOversight/pull/84